vagrant/provisioning/roles/arkcase-developer/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+- name: create arkcase database schema
+  include_tasks: "{{ role_path }}/../mariadb/tasks/create_schema.yml"
+  args:
+    apply:
+      vars:
+        item: arkcase
```

Comment:
Since we don't deploy ArkCase itself, we still have to create the database schema so ArkCase's Liquibase can do its thing.

Also, we don't deploy any Pentaho reports yet.  Still need to think through how to do this, since currently we deploy the reports from the arkcase.war.

Reply:
We can deploy the Pentaho reports at Arkcase startup, if they are not deployed. Also we can automatically update older reports and that would solve a lot of our issues.

Reply:
I thought about that... it's really the only way to know we are deploying the right reports.  We can scan for all reports in some certain folder, both in arkcase.war and in the extension jar.  We could send a JMS message, so the report deployment doesn't slow down the startup.